styles/scss/_articles.scss

```diff
@@ -1,39 +1,60 @@
 .section--articles {
-  padding: 50px;
-  margin: auto;
+  width: 90vw;
```

Comment:
@avtill to width: 90vw; jest tutaj niepotrzebne, ten kontener powinien rozciągać się na całą szerokość strony.